.github/sync-repo-settings.yaml

```diff
@@ -1,6 +1,6 @@
 # Whether or not rebase-merging is enabled on this repository.
 # Defaults to `true`
-rebaseMergeAllowed: true
+rebaseMergeAllowed: false
```

Comment:
Weird, I'm not sure how the script changed this, make sure this is desired.

Reply:
okay - reverting back this change.